metadata_service/config.py

```diff
@@ -29,6 +31,8 @@ class Config:
 
     PROXY_USER = os.environ.get('CREDENTIALS_PROXY_USER', 'neo4j')
     PROXY_PASSWORD = os.environ.get('CREDENTIALS_PROXY_PASSWORD', 'test')
+    PROXY_ENCRYPTED = True
```

Comment:
Was this off by default? If then, let's keep default `False`.

Reply:
This was previously on by default, and is on by default [in the driver](https://neo4j.com/docs/api/python-driver/1.7/driver.html#encrypted). Double checking the docs reminded me that I also want to be able to require certificate validation, which is the [trust](https://neo4j.com/docs/api/python-driver/1.7/driver.html#trust) parameter on the driver. My goal is to have full control over the TLS/SSL configuration. Should I include exposing the trust parameter in this PR?

Reply:
> This was previously on by default, and is on by default [in the driver](https://neo4j.com/docs/api/python-driver/1.7/driver.html#encrypted). Double checking the docs reminded me that I also want to be able to require certificate validation, which is the [trust](https://neo4j.com/docs/api/python-driver/1.7/driver.html#trust) parameter on the driver. My goal is to have full control over the TLS/SSL configuration. Should I include exposing the trust parameter in this PR?

@davcamer I see. It seems `encrypted` is `True` when TLS is enabled from their [doc](https://neo4j.com/docs/api/python-driver/1.7/driver.html#encrypted). I wonder what would happen when `TLS` is not enabled, but forcing `encrypted` to be `True`. Not sure if it's going to fail.

Reply:
I believe by "Defaults to True if TLS is available," they mean that it is supported by the python run-time that is executing the code. From browsing their code, this is the section that determines if it is available: https://github.com/neo4j-drivers/neobolt/blob/1.7/neobolt/compat/ssl.py#L23-L35

I don't see exactly where the conflicting configuration would be caught, so this could require some Amundsen users to update their config if they:
* Are running a client in a python runtime that doesn't support SSL.
* Configured their older Neo4j to accept _unencrypted_ bolt connections. Neo4j expects encrypted connections, even on older versions.

To preserve the existing behavior, I guess I could change this to set `encrypted = None` by default? This piece of code looks to expect a None, and automatically determine in that case:
https://github.com/neo4j/neo4j-python-driver/blob/1.7.6/neo4j/__init__.py#L226-L227

Reply:
@davcamer Sounds good to me. Let me know once you make a change. I tested out your branch and it works fine with me.

Reply:
I've made this change now. `PROXY_VALIDATE_SSL` is the option. It defaults to trust all certificates (no validation), since that was the previous behavior.